src/ethereum_test_forks/tests/test_forks.py

```diff
@@ -153,6 +161,50 @@ def test_forks():
     assert fork == Berlin
 
 
+def test_transition_fork_comparison():
+    """
+    Test comparing to a transition fork.
+
+    The comparison logic is based on the logic we use to generate the tests.
+
+    E.g. given transition fork A->B, when filling, and given the from/until markers,
+    we expect the following logic:
+
+    Marker    Comparison   A->B Included
+    --------- ------------ ---------------
+    From A    fork >= A    True
+    Until A   fork <= A    False
+    From B    fork >= B    True
+    Until B   fork <= B    True
+    """
+    assert BerlinToLondonAt5 >= Berlin
```

Comment:
Finally! does it also work for fork >= Cancun  and fork < Osaka ?

Reply:
I think `CancunToPrague >= Cancun` is a clear `True` IMO,  and also `CancunToPrague < Osaka`, but `PragueToOsaka < Osaka` is a bit harder to pin down.

Reply:
I'm going to keep an eye on new tests that make this kind of comparisons to see if it's worth it to handle comparing a transition fork to a normal fork, and we can make it stricter if necessary, but at this moment I think this should be ok.